drivers/ieee802154/Kconfig.nrf5

```diff
@@ -16,7 +16,6 @@ config IEEE802154_NRF5_DISABLED
 
 config IEEE802154_NRF5
 	bool "Regular mode"
-	select NET_L2_IEEE802154
```

Comment:
instead you should be able to do:
select NET_L2_IEEE802154 if not <ot l2 config>

Afaik. or is it ! instead of not? can't recall.